### PR TITLE
Add a tools section to the list

### PR DESCRIPTION
**Pull request by sijie**
- the Pulsar official dashboard
- pulsar-express by @bbonnin
- the official Pulsar grafana dashboard
- the grafana templates for Pulsar components

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,3 +47,15 @@ Other clients:
 - [logstash-input-pulsar](https://github.com/se7enkings/logstash-input-pulsar): A logstash input that receives data from Pulsar.
 - [pulsar-beat-output](https://github.com/streamnative/pulsar-beat-output): [Elastic Beats](https://github.com/elastic/beats) to Apache Pulsar.
 - [pulsar-flume-ng-sink](https://github.com/streamnative/pulsar-flume-ng-sink): An [Apache Flume](https://github.com/apache/flume) Sink implementation to publish data to Pulsar.
+
+## Tools
+
+### Dashboards
+
+- [Pulsar Dashboard](http://pulsar.apache.org/docs/en/administration-dashboard/): A web application that enables users to monitor current stats for all topics in tabular form.
```
Comment (Jennifer88huang-zz) [inline README.md]:
```suggestion
- [Pulsar Dashboard](http://pulsar.apache.org/docs/en/administration-dashboard/): A web application that enables users to monitor the current stats of all topics in the tabular form.
```

File: README.md [hunk 2]
```diff
@@ -47,3 +47,15 @@ Other clients:
...
+- [Pulsar Express](https://github.com/bbonnin/pulsar-express): Simple web interface for Apache Pulsar developed using [Nuxt.js](https://nuxtjs.org/) by [bbonnin](https://github.com/bbonnin).
```
Comment (Jennifer88huang-zz) [inline README.md]:
```suggestion
- [Pulsar Express](https://github.com/bbonnin/pulsar-express): A simple web interface for Apache Pulsar, it is developed with [Nuxt.js](https://nuxtjs.org/) by [Bruno Bonnin](https://github.com/bbonnin).
```

File: README.md [hunk 3]
```diff
@@ -47,3 +47,15 @@ Other clients:
...
+
+### Monitoring
+
+- [pulsar-grafana](http://pulsar.apache.org/docs/en/deploy-monitoring/#grafana): The pulsar grafana dashboard for Pulsar running on kubernetes.
```
Comment (Jennifer88huang-zz) [inline README.md]: - [Grafana](http://pulsar.apache.org/docs/en/deploy-monitoring/#grafana): With the `pulsar-grafana` Docker image, you can create a dashboard driven by the data stored in Prometheus. It is enabled by default when deploying Pulsar on Kubernetes.

File: README.md [hunk 4]
```diff
@@ -47,3 +47,15 @@ Other clients:
...
+- [apache-pulsar-grafana-dashboard](https://github.com/streamnative/apache-pulsar-grafana-dashboard): The grafana dashboard templates for different Pulsar components running in both kubernetes and on-prem machines.
```
Comment (Jennifer88huang-zz) [inline README.md]: - [Apache Pulsar Grafana Dashboard](https://github.com/streamnative/apache-pulsar-grafana-dashboard): Provides grafana dashboard templates for different Pulsar components running on both Kubernetes and on-premise machines.

**Review by Jennifer88huang-zz — APPROVED** [review]
Nit: You write it quite well, I just refine it from language and format perspective.

File: README.md [hunk 5]
```diff
@@ -48,7 +48,18 @@ Other clients:
...
+## Tools
+
+### Dashboards
+
+- [Pulsar Dashboard](http://pulsar.apache.org/docs/en/administration-dashboard/): A web application that enables users to monitor current stats for all topics in tabular form.
+- [Pulsar Express](https://github.com/bbonnin/pulsar-express): Simple web interface for Apache Pulsar developed using [Nuxt.js](https://nuxtjs.org/) by [bbonnin](https://github.com/bbonnin).
+
+### Monitoring
+
+- [pulsar-grafana](http://pulsar.apache.org/docs/en/deploy-monitoring/#grafana): The pulsar grafana dashboard for Pulsar running on kubernetes.
```
Comment (sijie) [inline README.md]:
```suggestion
- [pulsar-grafana](http://pulsar.apache.org/docs/en/deploy-monitoring/#grafana): With the `pulsar-grafana` Docker image, you can create a dashboard driven by the data stored in Prometheus. It is enabled by default when deploying Pulsar on Kubernetes.
```

File: README.md [hunk 6]
```diff
@@ -48,7 +48,18 @@ Other clients:
...
+- [apache-pulsar-grafana-dashboard](https://github.com/streamnative/apache-pulsar-grafana-dashboard): The grafana dashboard templates for different Pulsar components running in both kubernetes and on-prem machines.
```
Comment (sijie) [inline README.md]:
```suggestion
- [apache-pulsar-grafana-dashboard](https://github.com/streamnative/apache-pulsar-grafana-dashboard): Provides grafana dashboard templates for different Pulsar components running on both Kubernetes and on-premise machines.
```

**Comment by sijie** [comment]
@Jennifer88huang thank you for your review!

**Pull request merged**